servicetalk-concurrent-api/src/test/java/io/servicetalk/concurrent/api/ScanWithPublisherTest.java

```diff
@@ -390,13 +390,14 @@ public Integer mapOnComplete() {
                         }
 
                         @Override
-                        public boolean mapTerminal() {
+                        public boolean mapTerminal() throws InterruptedException {
                             if (interleaveCancellation) {
                                 checkpoint.countDown();
                                 try {
                                     resume.await();
                                 } catch (InterruptedException e) {
-                                    e.printStackTrace();
+                                    Thread.currentThread().interrupt();
+                                    throw e;
```

Comment:
Can we use `PlatformDependent.throwException(e)` instead of adding `throws InterruptedException` on API?